pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1550,5 +1554,55 @@ func TestAlgorithmProviderCompatibility(t *testing.T) {
 			}
 		})
 	}
+}
 
+func TestPluginConfigV1Alpha1(t *testing.T) {
+	// This test ensure the correctness of convert v1alpha1.PostFilter config to PreScore
+	testcases := []struct {
+		name        string
+		plugins     *config.Plugins
+		wantPlugins *config.Plugins
+	}{
+		{
+			name: "Test v1alpha1.PostFilter to config.PreScore",
+			plugins: func() *config.Plugins {
+				cfgv1alpha1 := v1alpha1.KubeSchedulerConfiguration{}
+				scheme.Scheme.Default(&cfgv1alpha1)
+				cfgv1alpha1.Plugins = &v1alpha1.Plugins{}
+				cfgv1alpha1.Plugins.PostFilter = &v1alpha1.PluginSet{
+					Enabled: []v1alpha1.Plugin{
+						{Name: "TestPostFilter1"},
+						{Name: "TestPostFilter2"},
+						{Name: "TestPostFilter3"},
+					},
+				}
+
+				cfg := config.KubeSchedulerConfiguration{}
+				if err := scheme.Scheme.Convert(&cfgv1alpha1, &cfg, nil); err != nil {
+					t.Fatal(err)
+				}
+				return cfg.Plugins
+			}(),
+			wantPlugins: func() *config.Plugins {
+				cfg := config.KubeSchedulerConfiguration{}
+				scheme.Scheme.Default(&cfg)
+				cfg.Plugins = &config.Plugins{}
+				cfg.Plugins.PreScore = &config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: "TestPostFilter1"},
+						{Name: "TestPostFilter2"},
+						{Name: "TestPostFilter3"},
+					},
+				}
+				return cfg.Plugins
+			}(),
+		},
+	}
+	for _, tc := range testcases {
+		t.Run(tc.name, func(t *testing.T) {
+			if !reflect.DeepEqual(tc.plugins, tc.wantPlugins) {
+				t.Errorf("config.diff:\n%s", diff.ObjectReflectDiff(tc.plugins, tc.wantPlugins))
```

Comment:
instead of importing the reflect pkg, just follow the same pattern used in the other test:

```
if diff := cmp.Diff(tc.wantPlugins, tc.plugins); diff != "" {
  t.Errorf("unexpected plugins diff (-want, +got): %s", diff)
}
```